src/sentry/lint/engine.py

```diff
@@ -128,8 +129,10 @@ def js_lint(file_list=None):
 
     has_errors = False
     if js_file_list:
-        status = Popen([eslint_path, '--config', eslint_config, '--ext', '.jsx', '--fix']
-                       + js_file_list).wait()
+        cmd = [eslint_path, '--config', eslint_config, '--ext', '.jsx', '--fix']
```

Comment:
You most certainly don't want to run `--fix` here but just exit if there is an error. Otherwise, you'd modify sources and probably even exit with a 0 status code.

Reply:
looks like this is actually a preexisting issue

Reply:
Got any context on why that was added? Looks odd to me...

Reply:
i made it respect the format param